accessibility/accessibility-in-pdfs.md

```diff
@@ -0,0 +1,92 @@
+# Accessibility in PDFs
+
+## A warning
+
+If you’re concerned about the accessibility of your PDF documents, then you should [consult a specialist like AbilityNet](https://www.abilitynet.org.uk/creating-accessible-pdfs) to review what you have, make expert recommendations, and potentially remediate issues for you. 
+
+## Is PDF the right choice?
+
+PDF is not the best document format for accessibility. It will always contain accessibility issues. PDF is a fixed dimension format intended for printing, and some user accessibility requirements --such as reflow, text resizing or changing text spacing-- are not supported in PDF format.
+
+We recommend that you avoid using PDF if you can, and prioritise accessible alternatives instead, such as HTML or [EPUB](https://www.w3.org/publishing/epub32/). As a reference, please read [why UK Government content should be published in HTML and not PDF](https://gds.blog.gov.uk/2018/07/16/why-gov-uk-content-should-be-published-in-html-and-not-pdf/).
+
+If you don’t have a choice, then you must purposefully build your PDFs to be accessible - **it doesn’t happen automatically**. 
+
+## Creating Accessible PDF Documents
```

Comment:
Other headings use sentence case instead of title case, which is more readable in my opinion.